cmd/commands/root.go

```diff
@@ -38,8 +39,10 @@ var RootCmd = &cobra.Command{
 	Args:             cobra.MinimumNArgs(1),
 	TraverseChildren: true,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		logging.Info("CephCluster namespace: %q", CephClusterNamespace)
```

Comment:
Before we create a new debug option, how about if we just comment out these two lines for now? Perhaps we don't even need a debug mode, or may use a different approach.